sdks/python/container/boot.go

```diff
@@ -200,25 +211,111 @@ func main() {
 		}
 	}
 
+	workerIds := append([]string{*id}, info.GetSiblingWorkerIds()...)
```

Comment:
@robertwb @chamikaramj do we have any tests that exercise the SiblingWorker  processes functionality? IIRC, this is new. @phoerious - is it correct that your current tests of Flink didn't test this?

Reply:
I have no idea how. Is this triggered if I allow multiple slots per taskmanager? I don't really understand the concept and would expect the runtime to simply start another worker on another port of needed.